Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -442,7 +442,11 @@ extension KMConversationViewController {
                     self?.showRatingView()
                     return
                 }
-                self?.show(feedback: previousFeedback)
+                if (self?.conversationClosedView.restartTapped != nil) {
+                    self?.showRatingView()
+                } else {
+                    self?.show(feedback: previousFeedback)
```

Comment:
Maybe we can remove this. Earlier we were showing feedback in the bottom when the conversation was rated, now we are asking for rating again. In the next sprint we'll take [this task](https://kommunicate.atlassian.net/browse/TD-1725) where we have to show the ratings in the chat thread. Currently, we do this in the web plugin.